app/scripts/filters/canI.js

```diff
@@ -9,8 +9,8 @@ angular
         {group: '', resource: 'buildconfigs/instantiate', verbs: ['create']}
       ],
       'builds': [
-        {group: '', resource: 'builds/clone', verbs: ['create']},
-        {group: '', resource: 'builds',       verbs: ['delete', 'update']}
+        {group: 'build.openshift.io', resource: 'builds/clone', verbs: ['create']},
+        {group: 'build.openshift.io', resource: 'builds',       verbs: ['delete', 'update']}
```

Comment:
Use `APIService.getPreferredVersion` here so it always matches what we use in the view.

Reply:
So `APIService.getPreferredVersion('builds')` -> `{group: 'build.openshift.io',     resource: 'builds' }`.  Do we want to take that & add the `verbs` property?

Example:
```JavaScript 
// update to this?
'builds': [
  _.extend(APIService.getPreferredVersion('builds/clone'), {verbs: ['create']});
  _.extend(APIService.getPreferredVersion('builds'), {verbs: ['delete', 'update']});
],
```

Reply:
Yes, except `_.extend` will mutate the preferred version object. Maybe put `verbs` first to avoid that problem.

```js
 _.extend({ verbs: ['create'], APIService.getPreferredVersion('builds/clone'));
```